docs/installation/others/enabling-temp-ssl-connection-in-ds.adoc

```diff
@@ -0,0 +1,160 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="enabling-temp-ssl-connection-in-ds_{context}"]
+
+// This content was copied and adjusted from https://github.com/dogtagpki/pki/wiki/Enabling-SSL-Connection-in-DS
+
+= Enabling Temporary SSL Connection in DS 
+
+*If you already have an active trusted CA, and you wish to issue a server cert for your DS, please follow link:enabling-ssl-connection-in-ds.adoc[this section] instead.*
+
+Follow this process using `pki` CLI (run `man pki-client`) commands to enable SSL connection in DS
+by creating a temporary DS bootstrap self-signed signing certificate and the bootstrap server certificate issued by it.
+
+This section assumes that a DS instance named `localhost` already exists,
+it does not have certificates, and the SSL connection is disabled.
+
+*Note:* In newer DS versions the certificates are created and the SSL connection is enabled by default,
+so in general it is not necessary to follow this procedure.
+
+== Creating DS Signing Certificate 
+
+First, generate DS signing CSR with the following command:
+
+----
+$ pki \
+    -d /etc/dirsrv/slapd-localhost \
+    -C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+    nss-cert-request \
+    --subject "CN=DS Signing Certificate" \
+    --ext /usr/share/pki/server/certs/ca_signing.conf \
+    --csr ds_signing.csr
+----
+
+Next, issue DS signing certificate:
+
+----
+$ pki \
+    -d /etc/dirsrv/slapd-localhost \
+    -C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+    nss-cert-issue \
+    --csr ds_signing.csr \
+    --ext /usr/share/pki/server/certs/ca_signing.conf \
+    --cert ds_signing.crt
+----
+
+Finally, import DS signing certificate:
+
+----
+$ pki \
+    -d /etc/dirsrv/slapd-localhost \
+    -C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+    nss-cert-import \
+    --cert ds_signing.crt \
+    --trust CT,C,C \
+    Self-Signed-CA
+----
+
+To verify the DS signing certificate:
+
+----
+$ certutil -L -d /etc/dirsrv/slapd-localhost -n Self-Signed-CA
+...
+    Certificate Trust Flags:
+        SSL Flags:
+            Valid CA
+            Trusted CA
+            User
+            Trusted Client CA
+        Email Flags:
+            Valid CA
+            Trusted CA
+            User
+        Object Signing Flags:
+            Valid CA
+            Trusted CA
+            User
+----
+
+== Creating DS Server Certificate 
+
+First, generate DS server CSR with the following command:
+
+----
+$ pki \
+    -d /etc/dirsrv/slapd-localhost \
+    -C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+    nss-cert-request \
+    --subject "CN=$HOSTNAME" \
+    --ext /usr/share/pki/server/certs/sslserver.conf \
+    --csr ds_server.csr
```

Comment:
Do we need SAN here?

Reply:
it's supposed to be a temporary one, but it's safer to add it just in case.  Thanks.